frontend/src/actions/steps/multi-step-action.ts

```diff
@@ -121,6 +124,8 @@ export class MultiStepAction {
         step.deactivate();
       }
     }
+
+    this.emit('completed');
```

Comment:
Wouldn't it be nice to also show a notification if a transfer fails?

Reply:
I didn't consider that as a part of the scope of this issue for the following reason:
- It requires some more thought process on what to display and how to fetch the right error messages so its somewhat human readable (as opposed to the current error messages)
- I consider this to also go hand in hand with this other big issue: https://github.com/beamer-bridge/beamer/issues/507

We could do a simple "Transfer failed" notificaiton but i am not sure how much benefit that will provide.

Your point is right though. As a user, if i figured out that the app shows a successful notification on transfer completion, i would also expect to get a notification for a tranfer failure as well.

Reply:
I think this can still be done separately from the mentioned issue. It is clear when a transfer is in a failed state (`failed` getter) and there will be an error message set (`errorMessage` getter). But well, it's true that not all error messages are user friendly at the moment and this needs improvement as the issue suggests.

So yeah let's not handle it in the scope of this PR but write an issue for it.